mlir/include/mlir/Dialect/EmitC/IR/EmitC.h

```diff
@@ -43,6 +43,16 @@ bool isIntegerIndexOrOpaqueType(Type type);
 
 /// Determines whether \p type is a valid floating-point type in EmitC.
 bool isSupportedFloatType(mlir::Type type);
+
+/// Determines whether \p type is a emitc.size_t/ssize_t type.
+bool isSizeTType(mlir::Type type);
```

Comment:
Naming is hard: Now that we renamed UnsignedSizeT to SizeT, this function name is odd. Maybe rename
`isSizeTType -> isAnySizeTType`?

Reply:
Let's do that, I don't have a better name.